net_printer/benchmarks.md

```diff
@@ -0,0 +1,27 @@
+# Benchmarks
+
+This page describes the suite of benchmark designs that are used to assess
+routing performance. The full list of benchmark designs, along with links to
+the original sources and some basic statistics is provided in the following
+table:
+
+|Source Benchmark Suite|Benchmark Name|LUTs|FFs|DSPs|BRAMs|
+|----------------------|--------------|----|---|----|-----|
+| [VTR](https://docs.verilogtorouting.org/en/latest/vtr/benchmarks/#vtr-benchmarks)|`mcml`|43k|15k|105|142|
```

Comment:
Can the benchmark name link to the specific benchmark please?

Reply:
`vtr_mcml`, `koios_dla_like_large`, and `rosetta_fd` are distributed as some kind of source code, so I can link to them directly. However, `ispd16_example2` is distributed as an archive of a `*.dcp` and I thought that linking directly to a download didn't seem appropriate. Finally, `boom_soc` was produced by generating RTL from the BOOM framework, so there isn't source code to link to. Should I link to the git repo instead of the webpage?

Reply:
It gets complicated to explain for now. Let's just leave it then.